style.css

```
@@ -1,3 +1,28 @@
+
+@font-face {
+    font-family: 'Inter';
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)